backend/src/main/java/org/sefglobal/core/service/CertificateService.java

```diff
@@ -0,0 +1,59 @@
+package org.sefglobal.core.service;
+
+import org.sefglobal.core.exception.ResourceNotFoundException;
+import org.sefglobal.core.model.Certificate;
+import org.sefglobal.core.repository.CertificateRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class CertificateService {
+
+    private final CertificateRepository certificateRepository;
+
+    public CertificateService(CertificateRepository certificateRepository) {
+        this.certificateRepository = certificateRepository;
+    }
+
+    /**
+     * Retrieves all the certificates in the repository
```

Comment:
According to the JavaDoc standards, you have to add a space after the description

Reply:
https://blog.joda.org/2012/11/javadoc-coding-standards.html

Reply:
On it!